src/System.Text.Json/src/System/Text/Json/JsonTokenType.cs

```diff
@@ -15,17 +15,68 @@ public enum JsonTokenType : byte
     {
         // Do not re-order.
         // We rely on the ordering to quickly check things like IsTokenTypePrimitive
+
+        /// <summary>
+        ///   Indicates that there is no value (as distinct from <see cref="Null"/>).
```

Comment:
I left some of comments regarding mentioning `null` `true` and `false` in triple slash comments, but I have the suspicion that you're referring to special objects created for the Json library.

If you are referring to a special object, then you should use `<see cref="Full DocId">`, but if you're using the C# reserved keywords, use `<see langword="keyword" />`.

Reply:
Based on previous discussion, this is not actionable (neither langword nor Full DocID).